Sources/PortalSwift/Core/Mpc/PortalMpcTypes.swift

```diff
@@ -14,61 +24,90 @@ public struct PortalMpcGeneratedShare: Codable, Equatable {
   }
 }
 
-/// A MPC share that includes a variable number of fields, depending on the MPC version being used
-/// GG18 shares will only contain: bks, pubkey, and share
-/// CGGMP shares will contain all fields except: pubkey.
+/// A MPC share that includes the share and optionally a backupSharePairId and a signingSharePairId.
 public struct MpcShare: Codable, Equatable {
```

Comment:
we shouldnt be touching these types. They should stay consistent across the update otherwise i dont think it will work with older shares

Reply:
It just makes some of the properties optional, we tested with v4 + v6 cross-device clients and they passed